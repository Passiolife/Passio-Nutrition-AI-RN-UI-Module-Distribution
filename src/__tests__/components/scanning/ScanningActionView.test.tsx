// @ts-nocheck
import React from 'react';
import renderer from 'react-test-renderer';
import {
  BrandingProvider,
  QuickScanningActionView,
  ServicesProvider,
} from '@passiolife/nutrition-ai-ui-ux';
import { mockBranding, mockServices } from '../../provider/MockProviders';
describe('ScanningActionView', () => {
  it('renders correctly', async () => {
    const onClosed = jest.fn();
    const tree = renderer.create(
      <ServicesProvider services={mockServices}>
        <BrandingProvider branding={mockBranding}>
          <QuickScanningActionView onClosedPressed={onClosed} />
        </BrandingProvider>
      </ServicesProvider>
    );
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
