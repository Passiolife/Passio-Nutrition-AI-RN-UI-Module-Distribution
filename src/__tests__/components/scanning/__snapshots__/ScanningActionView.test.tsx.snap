// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScanningActionView renders correctly 1`] = `
<View
  style={
    {
      "flex": 1,
      "height": 25,
      "left": 20,
      "position": "absolute",
      "top": 52,
      "width": 25,
    }
  }
>
  <View
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <Image
      resizeMode="contain"
      source={
        {
          "testUri": "../../../lib/commonjs/assets/ic_close_white.png",
        }
      }
      style={
        {
          "height": 24,
          "width": 24,
        }
      }
    />
  </View>
</View>
`;
